From 56cdea20f35b73be23fbc8099b41dd4aeece9634 Mon Sep 17 00:00:00 2001
From: "Yan, Shaoou" <shaopu.yan@intel.com>
Date: Thu, 8 Dec 2016 04:58:55 +0000
Subject: [PATCH 0489/2328] trusty: print out "Built: " in kernel directly.

do this instead of get them from trusty which can save 28
times vmexit/vmresume switch, so we can reduce some boot time

Change-Id: I196d506f606a77c1abe9a87d4d48dc18e40ca6bc
Signed-off-by: Feng, Wang <feng.f.wang@intel.com>
Reviewed-by: Ilkka Koskinen <ilkka.koskinen@intel.com>
---
 drivers/trusty/trusty.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/trusty/trusty.c b/drivers/trusty/trusty.c
index 8daf817..cfef965 100644
--- a/drivers/trusty/trusty.c
+++ b/drivers/trusty/trusty.c
@@ -401,7 +401,7 @@ static void trusty_init_version(struct trusty_state *s, struct device *dev)
 	}
 	s->version_str[i] = '\0';
 
-	dev_info(dev, "trusty version: %s\n", s->version_str);
+	dev_info(dev, "trusty version: Built: %s\n", s->version_str);
 
 	ret = device_create_file(dev, &dev_attr_trusty_version);
 	if (ret)
-- 
2.7.4

